addon/system/font-loaded.js

```diff
@@ -30,6 +25,34 @@ function injectAdobeBlank() {
   }
```

Comment:
Uh oh, this should have a `return true` at the end or the polling will never stop.